app/src/test/java/com/edulectronics/tinycircuit/ConnectorTest.java

```diff
@@ -0,0 +1,41 @@
+package com.edulectronics.tinycircuit;
+import com.edulectronics.tinycircuit.Models.Components.Connectors.ConnectionPoint;
+import com.edulectronics.tinycircuit.Models.Components.Connectors.ConnectionPointOrientation;
+import com.edulectronics.tinycircuit.Models.Components.Connectors.Connector;
+import com.edulectronics.tinycircuit.Models.Components.Lightbulb;
+import com.edulectronics.tinycircuit.Models.Components.Powersource;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static junit.framework.Assert.assertEquals;
+
+
+/**
+ * Created by Lennart on 12-12-2016.
+ */
+
+public class ConnectorTest {
+    @Test
+    public void connectTest(){
+        Lightbulb lightbulb = new Lightbulb();
+        Powersource powersource = new Powersource(5);
```

Comment:
lightbulb en powersource zouden van zichzelf al connectionpoints moeten hebben. Die zou je niet zomaar extra aan moeten kunnen maken of verwijderen. Moeten in de constructor gewoon gemaakt worden.